core/modules/utils/dom/animations/slide.js

```diff
@@ -37,6 +37,7 @@ function slideOpen(domNode,options) {
 			options.callback();
 		}
 	},duration);
+	$tw.anim.animationId = id;
```

Comment:
This is a red flag – `$tw.anim` is a shared instance that can be reused anywhere in the core, so we can't reach in and write to a property without interfering with other clients.